.github/workflows/snapshot.yml

```diff
@@ -90,3 +90,9 @@ jobs:
           echo "RELEASE_NAME=$RELEASE_NAME" >> $GITHUB_ENV
           echo "IS_PRE_RELEASE=$IS_PRE_RELEASE" >> $GITHUB_ENV
           echo "ARTIFACT_PATH=$PWD/$ARTIFACT" >> $GITHUB_ENV
+          echo "ARTIFACT_NAME=openocd-windows-${OPENOCD_TAG}" >> $GITHUB_ENV
+      - name: Publish OpenOCD packaged for windows
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ env.ARTIFACT_NAME }}
+          path: ${{ env.ARTIFACT_PATH }}
```

Comment:
@timsifive - If you'd like, the ARTIFACT_NAME improvement can be submitted also to the upstream so that there are less differences to maintain.

Reply:
Yes, although I'm not super concerned with this file since it's changed very rarely.